packages/atomic/src/components/commerce/product-template-components/atomic-product-children/atomic-product-children.tsx

```diff
@@ -144,7 +144,7 @@ export class AtomicProductChildren
         <img
           class="aspect-square p-1"
           src={this.getImageUrl(child)}
-          alt={child.ec_name}
+          alt={child.ec_name || undefined}
```

Comment:
can't help wondering whether a helper function to safely try to extract these things from an object would be useful - should we really be doing

```obj.something || underfined ```

or could we be doing something like

```getProductString(object, key)```

which would encapsulate this

Reply:
The TypeScript compiler is the safety here: Since everything is strict, and `alt` must be `string | undefined` (it's a basic HTML property), the compiler is doing the protection of `string | null does not map perfectly to string | undefined`, and refuse to build otherwise.

Reply:
I guess you don't feel the need to because the type definitions in `ChildProduct` force you to have those checks